[C++ Generator] Avoid convert string to char* warning

We should avoid warnings in generated C++ code so that we can activate -Werror.

In ValueEventsTest:
```
java.lang.RuntimeException: Compilation failed (exit status 1):
ValuedEventsTest.cc: In member function ‘virtual void {anonymous}::ValuedEventsTest_valuedEventsTest_Test::TestBody()’:
ValuedEventsTest.cc:48:56: error: ISO C++ forbids converting a string constant to ‘char*’ [-Werror=write-strings]
  statechart->getDefaultSCI()->raise_stringEvent( "tool");
                                                        ^
cc1plus: all warnings being treated as errors
```

--- a/test-plugins/org.yakindu.sct.generator.cpp.test/gtests/ValuedEventsTest/ValuedEventsTest.cc
+++ b/test-plugins/org.yakindu.sct.generator.cpp.test/gtests/ValuedEventsTest/ValuedEventsTest.cc
@@ -45,7 +45,7 @@ TEST_F(ValuedEventsTest, valuedEventsTest) {
 	
 	statechart->getDefaultSCI()->raise_realEvent( 20l);
 	
-	statechart->getDefaultSCI()->raise_stringEvent( "tool");
+	statechart->getDefaultSCI()->raise_stringEvent((sc_string) "tool");
 	
 	runner->proceed_cycles(1);
 	

